tools/starman.psgi

```diff
@@ -11,10 +11,15 @@ use Plack::Middleware::Static;
 my $app = LedgerSMB::PSGI::app();
 
 builder {
-   enable "Plack::Middleware::Static",
+    enable "Plack::Middleware::Static",
        path => qr{(^/?(images|doc|UI|css)/|favicon\.ico)}, root => '.';
-   mount '/stop.pl' => sub { exit; }
-	if $ENV{COVERAGE};
-   mount '/' => $app;
+
+    enable 'StackTrace', unsafe_ref_capture => 1;
+    enable "HTTPExceptions", rethrow => 1;
+
+    mount '/stop.pl' => sub { exit; }
+        if $ENV{COVERAGE};
```

Comment:
This is here specifically for coverage testing on Travis CI. If we have a development version which allows us to do exactly that, then maybe we shouldn't have this code in the production version?